test-proxy/src/main/java/com/google/cloud/bigtable/testproxy/CbtTestProxy.java

```diff
@@ -215,7 +215,11 @@ public synchronized void createClient(
       return;
     }
 
-    BigtableDataSettings.Builder settingsBuilder = BigtableDataSettings.newBuilder();
+    BigtableDataSettings.Builder settingsBuilder = BigtableDataSettings.newBuilder()
+        .setProjectId(request.getProjectId())
+        .setInstanceId(request.getInstanceId())
+        .setAppProfileId(request.getAppProfileId());
```

Comment:
what if the request doesn't have an app profile id?

Reply:
I also add the prefix "test(fix)" to the title and commit.